keyutil/src/main.rs

```diff
@@ -0,0 +1,47 @@
+use argh::FromArgs;
+use librad::keys::SecretKey;
+use std::fs;
+use std::fs::File;
+use std::io::prelude::*;
+use std::os::unix::fs::PermissionsExt;
+use std::path::Path;
+
+#[derive(FromArgs)]
+/// Utility for generating a private key file for Radicle Seed.
+pub struct Options {
+    /// filename of the generated key file (default: seed.key)
+    #[argh(option)]
+    pub filename: Option<String>,
```

Comment:
```suggestion
    pub filename: Option<PathBuf>,
```

Have you tried this instead? Saves you doing `Path::new` below